controller/user_test.go

```diff
@@ -248,6 +256,13 @@ func (m TestUserRepository) Load(ctx context.Context, id uuid.UUID) (*account.Us
 	return m.User, nil
 }
 
+func (m TestUserRepository) Exists(ctx context.Context, id string) (bool, error) {
+	if m.User == nil {
+		return false, errors.New("not found")
```

Comment:
No, you're right. The package is imported as is `"github.com/pkg/errors"` without the common `errs` alias. My bad. Sorry.